src/components/Text/index.tsx

```diff
@@ -47,6 +47,7 @@ const generateStyles = ({
   }` as keyof typeof themeColor
 
   return `
+    margin: 0;
```

Comment:
Where does the margin comes from ? Is this a browser stylesheet ? If this is a reset should it be put in normalize instead ?